rawfile/rawfile_servicer.py

```diff
@@ -204,14 +204,14 @@ def CreateVolume(self, request, context):
             )
 
         try:
-            params: dict[str, str] = {
-                k.lower(): v for k, v in request.parameters.items()
-            }
+            params = normalize_parameters(request.parameters.items())
             thin_provision = params.get("thinprovision", "no")
+            format_options = params.get("formatoptions", "")
             init_rawfile(
                 volume_id=request.name,
                 size=size,
                 thin_provision=str_to_bool(thin_provision),
+                format_options=format_options.split(" "),
```

Comment:
we don't need to add to the metadata, we can simply return them on the `volume_context` and then we receive the context in the stage call

Reply:
Yeah, just like lvm local pv